dockerfiles/Dockerfile.publisher

```diff
@@ -0,0 +1,27 @@
+# CI test image for unit/lint/type tests
+FROM node:lts-alpine
+
+RUN apk add --update bash git
+
+RUN rm -f ~/.npmrc
+
+ARG REG_BASIC_CREDENTIAL
+ARG REG_NPM_EMAIL
+RUN echo "_auth=$REG_BASIC_CREDENTIAL" >> ~/.npmrc
+RUN echo "email=$REG_NPM_EMAIL" >> ~/.npmrc
+RUN echo "always-auth=true" >> ~/.npmrc
+
+WORKDIR /app
+
+COPY package*.json ./
+RUN npm install
+
+COPY . .
+
+RUN npm run bootstrap
+RUN npm run build
+
+RUN git checkout .
+
+ARG PUBLISH_URL
+RUN ./node_modules/.bin/lerna publish --canary --yes --force-publish --registry $PUBLISH_URL --dist-tag $(git rev-parse --short HEAD) prerelease
```

Comment:
Do we need to heed this warning?

> If you have publish canary releases from multiple active development branches in CI, it is recommended to customize the --preid and --dist-tag <tag> on a per-branch basis to avoid clashing versions.
[source](https://github.com/lerna/lerna/tree/master/commands/publish#--canary)

It looks like the commit hash is used as the npm dist tag, not as the actual version of the package?